### ISSUE TITLE
[BUG]:live streaming one minute bars not subscribing correctly at open

### ISSUE THREAD

**Issue by TheHumanWithAPlan** [alpacahq/alpaca-trade-api-csharp#483]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I've emailed my issue to the support team multiple times but I don't think they understand the issue so I am posting here. I've tried to subscribe before market open ~9:27-9:29. This resulted in some minute bars being skipped, printed late or the server would just not send a message. I tried subscribing ~35 seconds after market opened and the same issue is seen. However subscribing to anytime after this, the WebSocket behaves normally. I have a screen capture video I can send the team to further demonstrate, let me know where to send. Thanks

### Environment

```markdown
- SDK Version: 5.2.9
- OS (version, bitness): Windows 11
- .NET SDK (version): 
- target process .NET version/bitness:4.7.2
```

**Comment by ooples** [comment]
I can help you with this issue. If you want send me some example code so I
can take a look where the issue is coming from or send me a message on
telegram @cheatcountry

**Comment by TheHumanWithAPlan** [comment]
> 

I've Tried to subscribe two ways
`var symb1 = client.GetMinuteBarSubscription("AAPL");
            await client.SubscribeAsync(symb1);
            symb4.Received += (trade) => { Console.Writeline(trade.Open);
                                                           };

   `
This works fine until I try to subscribe to more that 3 symbols, after the 3rd subscribed symbol no messages would be received. so I subscribe to all symbols and have an if loop to check the symbol. 

`var symb1 = client.GetMinuteBarSubscription("*");
            await client.SubscribeAsync(symb1);
            symb4.Received += (trade) => { 
                 if(trade.Symbol == "AAPL")
                    Console.Writeline(trade.Open);

                                                           };

   `

**Comment by OlegRa** [comment]
It sounds like a déjà-vu for me because I've already fixed a similar issue (#385) last year. I strongly recommend you to subscribe to`OnError` and `OnWarning` events and at least print exceptions/warnings to the console.

If you are OK with ALL symbols subscription I recommend keeping this approach - in this case, the subscription will work faster. And you can just call the `GetMinuteBarSubscription` method without arguments it will use wildcard by default.

Another option is to use the SDK extensions package for multi-symbol subscriptions. I think @ooples can provide you with examples for this approach.

**Comment by ooples** [comment]
@OlegRa You can assign me as I'm working with him on this bug

**Comment by TheHumanWithAPlan** [comment]
> It sounds like a déjà-vu for me because I've already fixed a similar issue (#385) last year. I strongly recommend you to subscribe to`OnError` and `OnWarning` events and at least print exceptions/warnings to the console.
> 
> If you are OK with ALL symbols subscription I recommend keeping this approach - in this case, the subscription will work faster. And you can just call the `GetMinuteBarSubscription` method without arguments it will use wildcard by default.
> 
> Another option is to use the SDK extensions package for multi-symbol subscriptions. I think @ooples can provide you with examples for this approach.

@OlegRa Subscribing to ```OnError``` always gives me "Invalid Syntax" in multiple versions of the package, I've added ```OnWarning``` now.  Why is ```GetMinuteBarSubscription``` to all symbols faster than individual symbol subscriptions?

**Comment by OlegRa** [comment]
If you need minute bars for ALL symbols wildcard subscription will work better. The client has to split the large list of subscribed symbols into smaller chunks and store a separate subscription object for each symbol. In the case of a wildcard subscription, it can send one small message and handle all incoming data using a single subscription object.
